src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Manifest/WorkloadManifest.targets.in

```diff
@@ -5,11 +5,13 @@
                                             '$(RuntimeIdentifier)' == 'browser-wasm' and
                                             '$(TargetFrameworkIdentifier)' == '.NETCoreApp' and
                                             !$([MSBuild]::VersionGreaterThanOrEquals('$(TargetFrameworkVersion)', '6.0'))">true</BrowserWorkloadDisabled>
+        <_AOTEnabled Condition="'$(RunAOTCompilation)' != ''">$(RunAOTCompilation)</_AOTEnabled>
+        <_AOTEnabled Condition="'$(_AOTEnabled)' == '' and '$(PublishAot)' != ''">$(PublishAot)</_AOTEnabled>
```

Comment:
If you have side-by-side `WorkloadManifest.targets` from .NET 6 and .NET 7, do we know if this file get imported twice?

Would a project import both:
```
C:\Program Files\dotnet\sdk-manifests\6.0.400\microsoft.net.workload.mono.toolchain\WorkloadManifest.targets
C:\Program Files\dotnet\sdk-manifests\7.0.100\microsoft.net.workload.mono.toolchain\WorkloadManifest.targets
```

Curious if we will need to deal with this in MAUI.